apis/Google.Cloud.Asset.V1/docs/history.md

```diff
@@ -1,5 +1,16 @@
 # Version history
 
+# Version 2.5.0-beta01, released 2020-09-24
+
+- [Commit 0ca05f5](https://github.com/googleapis/google-cloud-dotnet/commit/0ca05f5): Regenerate all APIs using protoc 3.13 and Grpc.Tools 2.31.
+
+  Note that the change in gRPC generation is *just possibly* breaking,
```

Comment:
We should decide whether or not we actually want to mention this at all. I still haven't thought of any cases where this would break code, even as corner cases, so I'm tempted to remove it entirely.

(If we do include it, I think it should be as a separate paragraph underneath the list, and probably shouldn't have as much detail. I'm happy to suggest specific text.)